src/kubernetes_api_objects/exec/object_meta.rs

```diff
@@ -186,6 +186,16 @@ impl ObjectMeta {
         self.inner.deletion_timestamp.is_some()
     }
 
+    #[verifier(external_body)]
+    pub fn well_formed(&self) -> (b: bool)
+        ensures b == self@.well_formed(),
+    {
+        self.inner.name.is_some()
+            && self.inner.namespace.is_some()
```

Comment:
K8s has cluster-wide objects which do not have a `namespace`, tho we currently do not support (and do not need) these objects. I would suggest you move `self.inner.namespace.is_some()` to each CR object's `well_formed`, and then you can leave a comment here to explain the decision.